### PR TITLE
manager: use rpc instead

### PR DESCRIPTION
**Pull request by Tricky511**

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -3,6 +3,5 @@
 // SPDX-License-Identifier: Apache-2.0
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    tonic_build::compile_protos("proto/manager.proto")?;
```
Comment (HHoflittlefish777) [inline build.rs:6]: It seems the file can delete？None  use in sealfs.

Comment (HHoflittlefish777) [reply build.rs:6]: https://github.com/labring/sealfs/blob/main/proto/manager.proto can also delete.

Comment (HHoflittlefish777) [reply build.rs:6]: https://github.com/labring/sealfs/blob/main/proto/fsbase.proto seems useless?I'm not sure

Comment (Tricky511) [reply build.rs:6]: `build.rs` and `manager.proto` have been deleted, but `fsbase.proto` does not know whether to delete

Comment (HHoflittlefish777) [reply build.rs:6]: @uran0sH @luanshaotong Is https://github.com/labring/sealfs/blob/main/proto/fsbase.proto useless？

File: src/rpc/client.rs [hunk 2]
```diff
@@ -356,7 +356,8 @@ impl ClientAsync {
                     .await;
                 match send_result {
                     Ok(_) => {
-                        let result = self.queue.wait_for_callback(id);
+                        let result =
+                            tokio::task::block_in_place(move || self.queue.wait_for_callback(id));
```
Comment (luanshaotong) [inline src/rpc/client.rs:360]: better to add a comment about future plans.

File: build.rs (hunk 1 above)
Comment (luanshaotong) [reply build.rs:6]: > `build.rs` and `manager.proto` have been deleted, but `fsbase.proto` does not know whether to delete

I think it is useless. Pls remove it.

Comment (HHoflittlefish777) [reply build.rs:6]: By the way，tonic dependency can delete

Comment (luanshaotong) [reply build.rs:6]: > By the way，tonic dependency can delete

Yes!

File: src/rpc/client.rs (hunk 2 above)
Comment (Tricky511) [reply src/rpc/client.rs:361]: done

File: build.rs (hunk 1 above)
Comment (Tricky511) [reply build.rs:6]: All about `tonic` deleted

**Pull request merged**